src/SavedParks/SavedParks.tsx

```diff
@@ -1,7 +1,42 @@
-const SavedParks: React.FC = () => {
+import { useEffect, useState } from 'react'
+import { LocalParkContainer, LocalParkData, AddRemoveFunctionality } from '../interfaces'
+import SavedCard from '../SavedCard/SavedCard'
+import { useLocation } from 'react-router-dom'
+
+const SavedParks: React.FC<LocalParkContainer & AddRemoveFunctionality> = props => {
```

Comment:
Nice solution for allowing us to pass multiple props with different interfaces to `SavedCard` component